.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: python
+python:
+  - 3.6
+cache:
+  directories:
+    - $HOME/.cache/pip
+    - $HOME/.ccache  # https://github.com/travis-ci/travis-ci/issues/5853
+
+install:
+  # Install this package and the packages listed in requirements.txt.
+  - pip install .
+  # Install extra requirements for running tests and building docs.
+  - pip install -r requirements-dev.txt
+
+env:
+   global:
+    # Doctr deploy key for NSLS-II-SIX/sixtools
```

Comment:
The mention of SIX seems like an accidental copy/paste stowaway.

Reply:
You are correct @danielballan this has been resolved in the latest commit